pinot-core/src/main/java/com/linkedin/pinot/core/common/DataBlockCache.java

```diff
@@ -184,6 +184,26 @@ public void initNewBlock(int[] docIds, int length) {
     return stringValues;
   }
 
+  /**
+   * Get byte[] values for the given single-valued column.
+   *
+   * @param column Column to read.
```

Comment:
(OCD) no period after 'param' or 'return'

Reply:
That's super OCD.